src/ansys/motorcad/core/geometry.py

```diff
@@ -526,8 +575,8 @@ class Entity(object):
 
     def __init__(self, start, end):
         """Initialise Entity."""
-        self.start = start
-        self.end = end
+        self.start = deepcopy(start)
```

Comment:
Trying to avoid people accidentally editing entities if they mutate a coordinate

@matthewAnsys thoughts?